src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -147,22 +147,25 @@ def rewrite_jsvm_lcov(self, zip_file_path):
             while not self.build_finished:
                 self.build_finished_cv.wait()
 
-        out_dir = zip_file_path[:-4]
+        out_name = zip_file_path[:-4]
+        out_dir = out_name
```

Comment:
Nit: `out_name` seems to be an unused alias of `out_dir`. Maybe remove it?

Reply:
Fixed with 43908596aacb03e7d7c2c40eca19a30dc7eb17d3.